#! /bin/sh /usr/share/dpatch/dpatch-run
# Description: Add Unity static quicklist support to inkscape
# Forwarded: https://bugs.launchpad.net/inkscape/+bug/676886
# Bug-Ubuntu: https://bugs.launchpad.net/bugs/676886
# Author: Didier Roche <didrocks@ubuntu.com>

@DPATCH@
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' inkscape-0.48.1~/inkscape.desktop.in inkscape-0.48.1/inkscape.desktop.in
--- inkscape-0.48.1~/inkscape.desktop.in	2010-07-13 05:48:40.000000000 +0200
+++ inkscape-0.48.1/inkscape.desktop.in	2011-02-17 16:21:42.690388423 +0100
@@ -12,3 +12,9 @@
 Terminal=false
 StartupNotify=true
 Icon=inkscape
+X-Ayatana-Desktop-Shortcuts=Drawing
+
+[Drawing Shortcut Group]
+_Name=New Drawing
+Exec=inkscape
+TargetEnvironment=Unity
